src/manual/tasks/scopes.md

```diff
@@ -0,0 +1,22 @@
+---
+filename: tasks/scopes.md
+title: Task Scopes
+order: 60
+---
+
+Taskcluster permissions are represented by "scopes", covered in more detail in
+[the design section](/manual/design/apis/hawk/scopes). For this section, it
+is enough to know that a scope is a permission to perform a particular action.
+
+A task has a set of scopes that determine what the task can do. For example, a
+worker may limit access to features based on scopes. It is also possible for a task
+to make arbitrary API calls using its scopes via an authenticating proxy.
+
+Taskcluster avoids "scope escalation", where a user might gain use of scopes
+they do not have, by requiring that the creator of a task have all of the
+scopes associated with the task. Thus a task's scopes are a subset of the scopes
+its creator has.
+
+Creating a task also requires the caller to have scopes for various parts of
+the task definition. The details are in the Queue's reference documentation,
+but this includes routes, priorities, and `schedulerId`.
```

Comment:
In other words, the all of the implicit things are handled by the queue, not by the worker - for example, docker-worker verifies there is a scope present for each cache, so there *is* a requirement to explicitly declare the scope in the task definition.  I think the same is true for generic-worker, right?